app/src/pages/admin/allUsersPage/allUsersPage.scss

```diff
@@ -0,0 +1,17 @@
+.header-buttons {
+  display: flex;
+  justify-content: flex-end;
+  button {

```

Comment:
I don't see any reasons to do it with the tag selector. Just wrap components in divs with corresponding classes.